BEGINNER_GUIDE.md

```diff
@@ -0,0 +1,167 @@
+Forking will create a copy of the repository in your Github account.
+![fork](https://user-images.githubusercontent.com/64194946/93016931-570b6500-f5e2-11ea-98fe-f226934091e4.jpg)
+
+
+**Step 2: Cloning**
+
+*For further steps, you need to have git installed. You can download git from [here](https://git-scm.com/downloads)*
+
+Now since you have git installed, run the following command to clone the repository: 
+```
+git clone https://github.com/YOUR_GITHUB_USER_NAME/PROJECT_REPOSITORY_NAME
+```
+*where YOUR_GITHUB_USER_NAME is your Github user name and PROJECT_REPOSITORY_NAME is the name of the project repository.*
+
+
+**Step 3: Remote**
+
+First, navigate to your local repository using:
+```
+cd PROJECT_REPOSITORY_NAME
+``` 
+
+The default remote name of the local repository is *origin*. This remote points to the forked repository. But you need to keep track of the main project repository. For this, we need to create a new remote named *upstream*. 
+
+Run the following command to add new remote:
+```
+git remote add upstream https://github.com/anitab-org/PROJECT_REPOSITORY_NAME
+```
+
+Use `git remote -v` to check the current remote. It should show the following output:
+```
+$ git remote -v
+origin  https://github.com/YOUR_GITHUB_USER_NAME/PROJECT_REPOSITORY_NAME (fetch)
+origin  https://github.com/YOUR_GITHUB_USER_NAME/PROJECT_REPOSITORY_NAME (push)
+upstream        https://github.com/anitab-org/PROJECT_REPOSITORY_NAME.git (fetch)
+upstream        https://github.com/anitab-org/PROJECT_REPOSITORY_NAME.git (push)
+``` 
+
+
+**Step 4: Pull latest changes from upstream**
+
+It is pretty obvious that their might be some changes in the main project after you have forked it. Your local repository should be synchronous to the main repository.
+
+For this, you have to pull changes from upstream master branch to your local repository.
+
+Run the following command to pull the latest changes:
+```
+git pull upstream master
+```
+
+
+**Step 5: Creating new branch**
+
+It is a good practice to a create separate branch for every issue so that it is isolated from the *master* branch.
+
+Use the following command to create a new branch and switch to it:
+```
+git checkout -b BRANCH_NAME
+```
+
+*Try to choose appropriate BRANCH_NAME. I would recommend using issue number as branch name (as it is unique to every issue).*
+
+
+**Step 6: Solving the issue**
+
+Now you can start working on the issue. You may need to read the installation guide that is specific to every project. For [VMS](https://github.com/anitab-org/vms) project you can check it's [installation guide](https://github.com/anitab-org/vms/blob/develop/aut_docs/Installation_Setup.md).
+
+Make the relevant changes that can fix the issue.
+
+***Stuck?***
+
+Don't worry! We are always here to help you. Go to the comment section of the issue and ask your doubt. Someone will help you out. You can also join our [ AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/) channel to discuss any topic.
```

Comment:
after any topic add 
`, for doubts, ideas and suggestions subscribe to the vms stream on zulip` (add link to stream)
as by default new users are not subscribed

Reply:
https://anitab-org.zulipchat.com/#streams/222539/vms is this link correct?

Reply:
yes correct one @ankit-saraf